operator/cmd/mesh/install.go

```diff
@@ -100,6 +102,13 @@ func InstallCmd(logOpts *log.Options) *cobra.Command {
   istioctl install --set "values.sidecarInjectorWebhook.injectedAnnotations.container\.apparmor\.security\.beta\.kubernetes\.io/istio-proxy=runtime/default"
 `,
 		Args: cobra.ExactArgs(0),
+		PreRunE: func(cmd *cobra.Command, args []string) error {
+			errs := validation.IsQualifiedName(iArgs.revision)
```

Comment:
I don't think `IsQualifiedName` is right here, you cannot set revision to `foo/bar`